compiler/tests/method_pointers.jm

```diff
@@ -0,0 +1,9 @@
+#HelloWorld\n
+
+def reprint(one:String)
+    print(one)
+end
+
+var = reprint
+
+print(var("HelloWorld"))
```

Comment:
reprint has no return type and thus a call to it cannot be used as an argument to print